add doc about how to align w macOS terminal keybinding (#5655)
microsoft/vscode-docs: docs/terminal/basics.md

* Ensure that you don't set `$EDITOR` to `vi(m)`. However, this isn't an option if you want your Git editor to work.
* Add `bindkey -e` to your init script to set Emacs explicitly.

> How can I configure a keybinding which maps `Cmd+C` to `Ctrl+C` in order to align with behavior in the macOS terminal?

Reviewer: Shouldn't this be JSON to put in keybinding.json for an end user?

Reviewer: This looks like the VS Code TypeScript code from src\vs\workbench\contrib\terminal\browser\terminal.contribution.ts.
Shouldn't this be a JSON snippet that an end user can add to their keybinding.json?

> How can I configure a keybinding which maps `Cmd+.` to `Ctrl+C` in order to align with behavior in the macOS terminal?

Reviewer 2: ```suggestion
### How can I configure `Cmd+.` to map to `Ctrl+C` like macOS' built-in terminal?
```

> ```json
> // Break: ctrl+C
> registerSendSequenceKeybinding(String.fromCharCode('C'.charCodeAt(0) - 64), {
> 	mac: { primary: KeyMod.CtrlCmd | KeyCode.Period }
> });
> ```

Reviewer 2: The macOS default terminal uses `kbstyle(cmd+.)` to perform the same as `kbstyle(ctrl+c)`. To get this behavior in VS Code, add this [custom keybinding](https://code.visualstudio.com/docs/getstarted/keybindings):

```suggestion
```json
{
	"key": "cmd+.",
	"command": "workbench.action.terminal.sendSequence",
	"when": "terminalFocus",
	"args": { "text": "\u0003" }
}
```
```

Reviewer 2: Opps I did the suggestion wrong, confusing because of multiple ``` blocks. It should say this before the json:

```
The macOS default terminal uses `kbstyle(cmd+.)` to perform the same as `kbstyle(ctrl+c)`. To get this behavior in VS Code, add this [custom keybinding](https://code.visualstudio.com/docs/getstarted/keybindings):
```
